tomviz/python/AutoTiltAxisRotationAlignment.py

```diff
@@ -87,8 +87,9 @@ def transform_scalars(self, dataset):
         rot_ang = coarseAngles[minIntensityIndex]
 
         self.progress.message = 'Rotating tilt series'
-        result = ndimage.interpolation.rotate(
-            tiltSeries, -rot_ang, axes=((0, 1)))
+        result = np.empty_like(tiltSeries)
```

Comment:
For scientific datasets we prefer to change the array size after rotation as doing so keeps the same unit.